src/tezos/TezosMessageCodec.ts

```diff
@@ -0,0 +1,242 @@
+import { TezosMessageUtils } from "./TezosMessageUtil";
+import { Operation } from "./TezosTypes";
+
+const operationTypes: Array<string> = [
+  "endorsement",
+  "seedNonceRevelation",
+  "doubleEndorsementEvidence",
+  "doubleBakingEvidence",
+  "accountActivate",
+  "proposal",
+  "ballot",
+  "reveal",
+  "transaction",
+  "origination",
+  "delegation"
+];
+
+export namespace TezosMessageCodec {
```

Comment:
Could we leave a comment about which kinds of functions this namespace contains?